src/components/index.ts

```diff
@@ -0,0 +1,3 @@
+import SingleAccordion from "./SingleAccordion/SingleAccordion.vue";
+
+export { SingleAccordion };
```

Comment:
Todo: Can we agree to use `Ris` as a prefix for components as per our [styleguide](https://digitalservicebund.atlassian.net/wiki/spaces/VER/pages/884310020/Frontend+conventions+and+patterns#Component-names-must-be-prefixed-with-Ris.)? This would make naming more consistent with the other projects.